modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheMapEntry.java

```diff
@@ -230,6 +231,10 @@
     @GridToStringExclude
     private final Lock listenerLock;
 
+    /** */
+    private final boolean disableTriggeringCacheInterceptorOnConflict =
```

Comment:
Let's make this 'static' otherwise we'll get noticeable performance drop due to HashTable lacks.